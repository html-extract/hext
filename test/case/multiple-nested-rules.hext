<*>
  {
    <p @text:content />
  }
  {
    <a href:href />
  }
</*>
