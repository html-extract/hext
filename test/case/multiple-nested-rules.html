<html>
  <body>
    <div> <p>P1 <a href="p1-href"></a></p> </div>
    <div> <p>P2 <a href="p2-href"></a></p> </div>
  </body>
</html>
