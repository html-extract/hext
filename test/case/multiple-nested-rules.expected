{"content":["P1","P2"],"href":["p1-href","p2-href"]}
{"content":["P1","P2"],"href":["p1-href","p2-href"]}
{"content":"P1","href":"p1-href"}
{"content":"P2","href":"p2-href"}
